mssql-plugin/src/main/java/io/cdap/plugin/mssql/SqlServerSourceSchemaReader.java

```diff
@@ -32,6 +32,7 @@ public class SqlServerSourceSchemaReader extends CommonSchemaReader {
   public static final int GEOMETRY_TYPE = -157;
   public static final int GEOGRAPHY_TYPE = -158;
   public static final int SQL_VARIANT = -156;
+  public static final int SMALLDATETIME_TYPE = 93;
```

Comment:
This is not required, you can directly user java.sql.Types.TIMESTAMP for matching

Reply:
done